mpd/playerstatus.go

```diff
@@ -47,4 +48,5 @@ func (p *PlayerStatus) Tick() {
 	diff := p.Since()
 	p.SetTime()
 	p.Elapsed += diff.Seconds()
+	p.ElapsedPercentage = float64(100) * p.Elapsed / float64(p.Time)
```

Comment:
I found that there was no error when no song was on the playlist and assumed (perhaps naively) that this meant there was no issue here. But I've added a check and amended.